cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
+	}
+
+	if len(argsWithProg[1]) == 0 {
+		panic("Given regex is empty!")
+	}
+
+	regex := argsWithProg[1]
+
+	_, err := os.Stdout.WriteString((generateCode(regex) + "\n"))
+	if err != nil {
+		panic("Error writing to stdout")
+	}
+}
+
+func generateCode(regex string) string {
+	afterLastBrace, beforeCurrentBrace, indentations, result := regex, "", "", "rex.New(\n"
```

Comment:
I think `result` should be either [strings.Builder](https://pkg.go.dev/strings#Builder) or [bytes.Buffer](https://pkg.go.dev/bytes#Buffer). Strings are immutable by mechanics, and each `+=` will create a new string. It is not bad in this scenario, because the number of characters is not big, but still it would be better to use.

Reply:
Ok, I will use [strings.Builder](https://pkg.go.dev/strings#Builder)